metadata.json

```diff
@@ -1,15 +1,21 @@
 {
   "name": "puppet-misp",
-  "version": "0.2.0",
+  "version": "0.2.3-rc0",
```

Comment:
That was dont by the travis_release. I understood that I could executre the rake but its true I didnt have permsisions.